.github/workflows/test262.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       matrix:
         shard:
           # Ensure SHARD_TOTAL is updated with the number of shards declared here
-          [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 18, 19]
+          [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
```

Comment:
20 was already done in 1:30. 10 is reasonable and should have less chance of causing 429  on checkout.